modyn/config/schema/downsampling_config.py

```diff
@@ -0,0 +1,145 @@
+from __future__ import annotations
+
+from typing import Annotated, List, Literal, Union
+
+from modyn.config.schema.modyn_base_model import ModynBaseModel
+from pydantic import Field, model_validator
+from typing_extensions import Self
+
+
+class BaseDownsamplingConfig(ModynBaseModel):
```

Comment:
Ahh, that looks much cleaner and is way more user-friendly if someone isn't familiar with the configuration for sampling.